recipe/patches/0020-Don-t-checksharedmods-if-cross-compiling.patch

```diff
@@ -24,17 +24,12 @@ index 689ece9d24..913cbd8e23 100644
  rundsymutil: sharedmods $(PYTHON_FOR_BUILD_DEPS) $(BUILDPYTHON)
 diff --git a/if_runnable.sh b/if_runnable.sh
 new file mode 100755
-index 0000000000..14f84a8597
+index 00000000000..14f84a8597c
 --- /dev/null
 +++ b/if_runnable.sh
-@@ -0,0 +1,10 @@
+@@ -0,0 +1,5 @@
 +#!/bin/bash
 +
 +if [[ "${target_platform}" == "${build_platform}" || "${CROSSCOMPILING_EMULATOR:-}" != "" ]]; then
-+  if [[ "$1" == "./"* ]]; then
-+    exec $@
-+  else
-+    export ${@:1:$#-2}
-+    exec ${@: -2}
-+  fi
++    exec $*
```

Comment:
@h-vetinari This diff looks at fault.

Reply:
Strange, I didn't touch that patch when rebasing, that was git's doing.

Reply:
Ah, nevermind, the reason was that I hadn't picked the fixes from #696 into my local branch 🤦